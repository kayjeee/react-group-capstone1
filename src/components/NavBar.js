import React from 'react';
import { NavLink } from 'react-router-dom';
import logo from '../img/logo.png';
import './Header.css';

export default function NavBar() {
  return (
    <header className="flex">
      >
      <div className="logo-details flex">
        <img src={logo} alt="" />
        <h1>Space Traveler&apos;s Hub</h1>
      </div>
      <nav>
        <ul>
          <li>
            <NavLink to="/" className="menu-item">Rockets</NavLink>
          </li>
          <li>
            <NavLink to="/missions" className="menu-item">Missions</NavLink>
          </li>
          <li>
            <NavLink to="/profile" className="menu-item">My Profile</NavLink>
          </li>
        </ul>
      </nav>
    </header>
  );
}
