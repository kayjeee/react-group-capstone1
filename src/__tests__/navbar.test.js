import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import NavBar from '../components/navBar';

test('make snapshots on the nav', () => {
  const tree = renderer.create(<MemoryRouter><NavBar /></MemoryRouter>).toJSON();
  expect(tree).toMatchSnapshot();
});
