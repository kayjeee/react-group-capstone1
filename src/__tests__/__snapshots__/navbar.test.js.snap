// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`make snapshots on the nav 1`] = `
<header
  className="flex"
>
  <div
    className="logo-details flex"
  >
    <a
      aria-current="page"
      className="active"
      href="/"
      onClick={[Function]}
    >
      <img
        alt="planets"
        className="img-logo"
        src="planet.png"
      />
    </a>
    <h2>
      Space Travelersâ€™ Hub
    </h2>
  </div>
  <ul
    className="list-container flex"
  >
    <li>
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        Rockets
      </a>
    </li>
    <li>
      <a
        className=""
        href="/missions"
        onClick={[Function]}
      >
        Missions
      </a>
    </li>
    <li>
      <a
        className=""
        href="/myprofile"
        onClick={[Function]}
      >
        My Profile
      </a>
    </li>
  </ul>
</header>
`;
